src/rt/frun.cc

```diff
@@ -99,14 +99,25 @@ trip_idx_t run_stop::get_trip_idx(event_type const ev_type) const {
       .at(0);
 }
 
-std::string_view run_stop::trip_display_name(event_type const ev_type) const {
+std::string_view run_stop::route_short_name(event_type const ev_type) const {
   if (fr_->is_rt() && rtt() != nullptr) {
-    return rtt()->transport_name(tt(), fr_->rt_);
+    return rtt()->route_short_name(tt(), fr_->rt_);
   }
   if (fr_->is_scheduled()) {
-    return tt().trip_display_names_[get_trip_idx(ev_type)].view();
+    return tt().route_short_name(get_trip_idx(ev_type));
   }
-  return "?";
+  return {};
+}
+
+std::string_view run_stop::trip_short_name(
+    event_type const event_type) const noexcept {
```

Comment:
```suggestion
    event_type const event_type) const {
```

I guess `noexcept` is correct here but since we had several crashes where I thought no exception can be raised and MOTIS terminated completely, I decided to not use `noexcept` except it's in hot loops and improves performance.

Reply:
Ah yes, I just messed up the rebase here.